public/v4/apps/audiobookshelf.yml

```diff
@@ -19,9 +19,19 @@ caproverOneClickApp:
       description: Check out their Docker page for the valid tags https://hub.docker.com/r/advplyr/audiobookshelf/tags
       validRegex: "/.{1,}/"
 
-    - id: $$cap_path_to_audiobookshelf
-      label: Location of app folder in host
-      description: path to base folder that will contain your audiobooks, podcasts, metadata and config.
+    - id: $$cap_path_to_audiobooks
+      label: Location of audiobooks directory
+      description: full path to the directory where the audiobooks are stored
```

Comment:
Make it explicit that this path MUST EXISTS on the host, otherwise the app won't start.

Same with the other paths.